### PR TITLE
Advance gRPC version to 1.1.0-pre1

### PR DESCRIPTION
**Pull request by muxi**
cc: @nicolasnoble @hsaliak 

Podspecs will be submitted to CocoaPods after release tag is created.

### CODE REVIEW

**Comment by muxi** [comment]
Mac test is expected to fail before release is tagged.

File: src/objective-c/BoringSSL.podspec [hunk 1]
```diff
@@ -900,588 +897,585 @@ Pod::Spec.new do |s|
...
-          "SECTION_NOT_FOUND\\0"
-          "UNABLE_TO_GET_ISSUER_DETAILS\\0"
-          "UNABLE_TO_GET_ISSUER_KEYID\\0"
-          "UNKNOWN_BIT_STRING_ARGUMENT\\0"
-          "UNKNOWN_EXTENSION\\0"
-          "UNKNOWN_EXTENSION_NAME\\0"
-          "UNKNOWN_OPTION\\0"
-          "UNSUPPORTED_OPTION\\0"
-          "USER_TOO_LONG\\0"
+          "ASN1_LENGTH_MISMATCH\0"
```
Comment (jcanizales) [inline src/objective-c/BoringSSL.podspec]: You need double slash here (`\\`), or Ruby will interpret it as a `0` byte.

File: gRPC-Core.podspec [hunk 2]
```diff
@@ -35,7 +35,7 @@
 
 Pod::Spec.new do |s|
   s.name     = 'gRPC-Core'
-  version = '1.0.2'
+  version = '1.1.0-pre1'
```
Comment (jcanizales) [inline gRPC-Core.podspec]: This is as good a time as ever to get the version number from a template :)

Comment (muxi) [inline gRPC-Core.podspec]: that is right... I am thinking of that too. Not only the version number but automation of the entire advancement process

**Pull request merged**